src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -121,6 +121,13 @@ class HtmlAudioPlayer {
                     normalizationGain =
                         options.mediaSource.albumNormalizationGain
                         ?? options.item.NormalizationGain;
+                } else {
+                    console.debug('normalization disabled');
+                    return;
+                }
+
+                if (!self.gainNode) {
+                    addGainElement(elem);
```

Comment:
Probably missing from the beginning.
```suggestion
                    addGainElement(elem);
                    if (!self.gainNode) return;
```
`addGainElement` already has an error message.